spec/support/act_as_matcher.rb

```diff
@@ -0,0 +1,44 @@
+require 'rspec/expectations'
+
+RSpec::Matchers.define :act_as do |actable|
```

Comment:
Unused block argument - `actable`. You can omit the argument if you don't care about it.